contrib/connector/nvmeof/nvmeof.go

```diff
@@ -22,7 +22,6 @@ type Nvmeof struct{}
 
 func init() {
 	connector.RegisterConnector(connector.NvmeofDriver, &Nvmeof{})
-	connector.ExecCmd("modprobe", "nvme-rdma")
```

Comment:
Can u explain why u remove this line?

Reply:
Redundant code, already doing similar actions elsewhere.